client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html

```diff
@@ -4,7 +4,16 @@
         <tuc-section-back-link data-tuc-section-back-link-to-state="telecom.telephony.line.calls"
             data-tuc-section-back-link-title="{{ 'telephony_line_calls_back_link' | translate }}">
         </tuc-section-back-link>
-        <h1 data-translate="telephony_line_actions_line_calls_call_waiting_title"></h1>
+        <h1>
+            {{:: 'telephony_line_actions_line_calls_call_waiting_title' | translate }}
```

Comment:
Prefer using `ngBind` over `{{` (`data-translate`)

Reply:
I'm using directly the `translate` directive.

```html
<span data-translate="telephony_line_actions_line_calls_call_waiting_title"></span>
```

Reply:
Thanks for the explanation, I wasn't sure.